docs/tutorial/kymotracking.rst

```diff
@@ -296,6 +328,20 @@ The independently determined offset (in photons per pixel) can then be provided
 .. image:: figures/kymotracking/gaussian_refined_offset.png
 
 In this case the parameter will not be fitted, but fixed to the user specified value. This can help reduce the variance of the parameter estimates.
+Especially the estimates for the photon counts will benefit from doing an offset estimation prior to fitting::
```

Comment:
Right. I think I should be more specific here. The precision will go up. The accuracy may or may not go down depending on whether it is representative.
```
Estimating an offset independently prior to fitting can improve the precision of the estimates (since it requires fewer parameters to be estimated for each window).
```